### ISSUE TITLE
self.graph_out can be None

### ISSUE THREAD

**Issue by ihopethiswillfi** [timeseriesAI/tsai#240]
I've copy-pasted the Minirocket tutorial code and I'm running it locally in a .py file on a Linux machine.
I'm running the very latest commit of tsai.

When executing
`learn.fit_one_cycle(10, 3e-4)`
I get `'NoneType' object has no attribute 'update'`

The responsible line is `self.graph_out.update(self.graph_ax.figure)` in core.py

I'm unfortunately unable to debug this currently but a quick fix was to simply insert a line above it: `if self.graph_out:`

**Comment by oguiza** [comment]
Hi,
I'm sorry for my late reply!
Could you please paste the full trace? Otherwise, I don't know where the issue is coming from.

**Comment by ihopethiswillfi** [comment]
Hey Ignacio, in the mean time I've moved on to another project unfortunately.  I'm pretty sure that at the time, this was caused by me running it from inside a .py file instead of a notebook.  From what I can remember, the `display()` function seemed to return None when it's not run from inside a notebook.
I hope that helps, if not, feel free to close the issue.
Great job by the way, this is an impressive project.
